garth/users/settings.py

```
@@ -2,6 +2,7 @@
 from typing import Dict, List
 
 from pydantic.dataclasses import dataclass
+from typing import List, Optional
```

Ruff (0.8.2)

Redefinition of unused `List` from line 2

Remove definition: `List`

(F811)